app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -299,14 +299,16 @@
                 = _("Disable event monitoring.")
       = miq_tab_content('ssh_keypair', 'default') do
         .form-group
-          .col-md-12.col-md-12{"ng-if" => "#{ng_model}" == "emsCommonModel" && "#{ng_model}.emstype == 'openstack_infra'"}
+          .col-md-12.col-md-12{"ng-if" => "#{ng_model}" == "emsCommonModel" && ("#{ng_model}.emstype == 'openstack_infra' || #{ng_model}.emstype == 'rhevm'") }
             = render :partial => "layouts/angular-bootstrap/endpoints_angular",
                                  :locals  => {:ng_show                => true,
                                               :ng_model               => "#{ng_model}",
                                               :id                     => record.id,
                                               :hostname_hide          => true,
                                               :apiport_hide           => true,
                                               :security_protocol_hide => true,
+                                              :tls_verify_hide        => true,
+                                              :tls_ca_certs_hide      => true,
```

Comment:
@AparnaKarve will this be ok for openstack or should it be true only for rhv?

Reply:
@borod108 It should be OK for OpenStack -- although, it looks like for both OpenStack and RHV, we do not need the partial - "layouts/angular-bootstrap/endpoints_angular",  since none of the partial elements (hostname, api, security_protocol) are being used for `ssh_keypair`